src/components/app/details/main.tsx

```diff
@@ -104,6 +104,20 @@ export function AppHeader({ appName }: { appName: string }) {
     const location = useLocation()
     const currentPathname = useRef('')
 
+    function trackEvent(category: string, action: string): void {
+        ReactGA.event({
+            category: category,
+            action: action,
+        });
+    }
+
+    function preventLinkDefault(event: React.MouseEvent<Element, MouseEvent>, className: string) {
```

Comment:
Can we please rename this to a bit more precisely something like `onClickTabPreventDefault`